### PR TITLE
my second task

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: cartaya/2/style.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+}
+
+#lupa {
+    width: 24px;
+    height: 27px;
+    float: left;
+    position: relative;
+}
+
+#people-search {
```
Comment (reiniergs) [inline cartaya/2/style.css]: you shouldn't never ever use css selector for element id, use classes

File: cartaya/2/style.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+#boton:hover{
+    background-color: rgb(167, 167, 167);
+    transition: background-color .3s;
+}
+
+.selector-azul{
+    -webkit-appearance: checkbox;
+}
+label {
```
Comment (reiniergs) [inline cartaya/2/style.css]: you shouldn't never ever use css selector for element name, use classes ... there reason behind this is that when you apply styles directly to an element, it means all element label in you website have applied the same styles

**Review by reiniergs — CHANGES_REQUESTED** [review]

**Comment by reiniergs** [comment]
fix the conflits as well

File: cartaya/2/task2.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        <nav>
+            <form action="#" method="get" id="people-search">
+                <div id="lupa" class="color-borde">
+                    <img  class="lupa" src="lupa.png" width="15px" height="15px" alt="search">
+                </div>
+               
+                <input id="search" type="search" name="people-search" placeholder="search" aria-label="search throught site content" class="color-borde">
+                <input id="boton" type="submit" value="Search"> <br /> <br />
+                
+                <input class="selector-azul" checked type="radio" id="last-name" name="people-search" value="last-name" aria-label="last name">
```
Comment (TahimiLeonBravo) [inline cartaya/2/task2.html]: You need always use a convention to the names of all classes, it should create collisions.

File: cartaya/2/task2.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+               
+                <input class="selector-azul" type="radio" id="first-name" name="people-search" value="first-name" aria-label="first name">
+                <label for="first-name">First Name</label>
+                
+                <input class="selector-azul" type="radio" id="department" name="people-search" value="department" aria-label="department">
+                <label for="department">Department</label>
+            </form>
+        </nav>
+
+        <section class="perfil">
```
Comment (TahimiLeonBravo) [inline cartaya/2/task2.html]: You need always use a convention to the names of all classes, it should create collisions.

File: cartaya/2/style.css [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+#perfil3 {
+    margin-left: 0px;
+    margin-right: 32px;
+}
+
+.arrow:hover {
+    cursor: pointer;
+    transform: scale(1.4,1.4);
+    transition: transform .3s;
+}
```
Comment (TahimiLeonBravo) [inline cartaya/2/style.css]: You have to add a newline at the end of the file

File: cartaya/2/task2.html [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        <div class="arrow">
+            <img src="flecha delante.png" height="51px" width="30px" alt="next">
+        </div>
+
+        <div class="arrow">
+            <img src="flecha atras.png" height="51px" width="30px" alt="back">
+        </div>
+    </main>
+ </body>
+</html>
```
Comment (TahimiLeonBravo) [inline cartaya/2/task2.html]: You have to add a new line at the end of the file

File: cartaya/2/task2.html [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This is my second task from Leo">
+    <meta name="keywords" content="study,html5,css3,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <link rel="stylesheet" href="style.css">
+    <title>Task#2</title>
+</head>
+ <body>
+    <header>
+        <span id="h1">Intranet</span>
```
Comment (TahimiLeonBravo) [inline cartaya/2/task2.html]: Never use the attribute id for elements name, use class. the id is a global attribute that defines a unique identifier which must be unique in the whole document.

File: cartaya/2/style.css [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+    margin-left: 53px;
+}
+
+#search {
+    width: 213px;
+    height: 32.33px;
+    padding: 1px 1px 1px 2px;
+    float: left;
+    position: relative;
+    font-family: HelveticaNeueCE35Thin;
```
Comment (TahimiLeonBravo) [inline cartaya/2/style.css]: You can define the font-family property in the element container, this is an Inherited property you don’t need specify in each element inside the container. If all the texts in the document have the same font, define this property only in the general element container.

**Review by TahimiLeonBravo — APPROVED** [review]

**Review by TahimiLeonBravo — APPROVED** [review]

**Review by TahimiLeonBravo — APPROVED** [review]

**Pull request closed without merging**